### ISSUE TITLE
naughty.notify only showing first few lines

### ISSUE THREAD

**Issue by coldfix** [awesomeWM/awesome#2844]
Output of `awesome --version`:

```awesome v4.3 (Too long)
 • Compiled against Lua 5.3.5 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2
```

**How to reproduce the issue:**

Send a multi-line notification via naughty, e.g.

```bash
awesome-client 'require("naughty").notify { text="a\nb\nc\nd\ne\nf\n" }'
```

**Actual result:**

Shows notification with only the first few lines (in this case lines a to d, but not the remaining lines).

**Expected result:**

Shows notification with full text.

This affects both my archlinux machines. Tried it both with my customized config as well as the default config in `/etc/xdg/awesome/rc.lua`. I noticed because this broke my calendar widget, this is relatively new (maybe a week?). This is probably related to the update of some other software package, since the awesome package is unchanged since February 2019, and I'm sure the issue started appearing not more than 1 or two weeks ago. Packages updated since then:

Not sure if this is relevant, but the most recent updates were:

```
This week:
glib2 (2.60.5-1 -> 2.60.6-1)
cmake (3.15.0-1 -> 3.15.1-1)
docker (1:18.09.8-1 -> 1:19.03.1-1)
fuse-common (3.6.1-1 -> 3.6.2-1)
fuse3 (3.6.1-1 -> 3.6.2-1)
glib2-docs (2.60.5-1 -> 2.60.6-1)
glu (9.0.0-5 -> 9.0.1-1)
gnutls (3.6.8-3 -> 3.6.9-1)
imagemagick (7.0.8.56-1 -> 7.0.8.57-1)
lib32-glib2 (2.60.5-1 -> 2.60.6-1)
pango (1:1.43.0-2 -> 1:1.44-1)
lib32-pango (1:1.43.0-2 -> 1:1.44-1)
libxfce4util (4.13.4-1 -> 4.13.5-1)
wayland-protocols (1.17-1 -> 1.18-1)
libxfce4ui (4.13.6-1 -> 4.13.7-1)
linux (5.2.2.arch1-1 -> 5.2.3.arch1-1)
python-lxml (4.3.4-1 -> 4.4.0-1)
python-pyrsistent (0.15.3-1 -> 0.15.4-1)
srt (1.3.2-1 -> 1.3.3-1)
udisks2 (2.8.3-1 -> 2.8.4-1)
xterm (347-1 -> 348-1)
```

**Comment by coldfix** [comment]
I had a hunch it might be related to the pango update and can now confirm that the issue is fixed by downgrading pango to `1:1.43` and restarting awesome and appears again by upgrading to `1:1.44`.

So this might be a pango bug (or bugfix, who knows), but I don't have the insight to compile a bug report for them.

**Comment by psychon** [comment]
Uhm... okay. That means at least for me that I cannot do much about this (debian testing).

Thanks for figuring out where the problem comes from!

**Comment by necauqua** [comment]
I confirm same behavior on my arch machine, thank you for the workaround (downgrading pango)

**Comment by coldfix** [comment]
Issue is resolved with pango update to `1:1.44.3`.

**Comment by Elv13** [comment]
I pinned this until we (or they) can find a solution and get it pushed into Arch/Gentoo/DebianSid. I guess it will be a common occurrence in the next couple weeks.

**Comment by coldfix** [comment]
The issue was solved with either pango 1.44.2 or 1.44.3 which is already in Arch.

**Comment by ViSaturn** [comment]
Why is this still pinned.
